### ISSUE TITLE
VoiceOver does not announce missing required field; "Create Organization Unit" button remains enabled

### ISSUE THREAD

**Issue by vanpham-sw** [shopware/shopware#7823]
### PHP Version

8.2.25

### Shopware Version

6.7

### Affected area / extension

Extension:Commercial

### Actual behaviour

When attempting to create an organization unit using VoiceOver, if the user forgets to select a required field, the "Create Organization Unit" button remains enabled. Additionally, VoiceOver does not provide any auditory feedback to inform the user that a required field is missing. This leads to confusion, as users are not alerted to correct the input before submitting the form.

### Expected behaviour

VoiceOver should announce a message indicating that a required field has not been selected and the "Create Organization Unit" button should be disabled until all required fields are completed.

### How to reproduce

1. Enable VoiceOver on the device.
2. Navigate to the "Create Organization Unit" page in the application.
3. Fill in some fields but leave at least one required field unselected.
4. Attempt to click the "Create Organization Unit" button.

**Comment by vanpham-sw** [comment]
Hi @yusufttur,

We're seeking your advice on a part of the bug related to the "Create Organization Unit" button, which remains enabled. This mirrors the current form behavior elsewhere (such as in create role, employee, address).

I suggest that the voice-over provides auditory feedback to inform the user when a required field is missing upon clicking the "Create Organization Unit" button, is enough and we can keep the button enabled. 
However, I believe your input is crucial here, so could you please take a look? Thank you!

**Comment by ravenokko** [comment]
> I suggest that the voice-over provides auditory feedback to inform the user when a required field is missing upon clicking the "Create Organization Unit" button, is enough and we can keep the button enabled.

As a general rule buttons for submit forms should remain enabled. See: 
https://github.com/shopware/shopware/issues/12432#issuecomment-3861071633
https://adrianroselli.com/2024/02/dont-disable-form-controls.html

So I would agree the auditory feedback is sufficient.

**Comment by yusufttur** [comment]
I found out that `create unit `button on the existing page don't receive any focus when pressing `tab` button. Can be fixed under this issue.

<img width="1215" height="312" alt="Image" src="https://github.com/user-attachments/assets/26403666-440c-45c2-8df0-2c7836e7c14b" />
